src/components/timeline/RowYAxes.svelte

```diff
@@ -146,10 +146,8 @@
         }
       }
 
-      totalWidth += marginWidth;
```

Comment:
Think we need to keep track of total width here for the purposes of drawing multiple axes on one row